src/store/reducers/apiResponse.reducer.js

```diff
@@ -16,7 +16,7 @@ export const confirmationModalState = {
     needConfirm: false,
     status: null,
     title: '',
-    onConfirm: null,
+    onConfirm: () => {},
```

Comment:
why does this now about a possible action?

Reply:
I'm not sure that I've understood your question correct, @Vxplus . But, I've made this change because while it was initially `null` - it was triggering some harmless, but still **error**(during `prop-types` checking) in the console, because of it `isRequired` property, but initially it was null.